SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -330,6 +352,13 @@ bool FillSnapshot(Worker_SnapshotOutputStream* OutputStream, UWorld* World)
 			   UTF8_TO_TCHAR(Worker_SnapshotOutputStream_GetState(OutputStream).error_message));
 		return false;
 	}
+	
+	if (!CreateRoutingWorkerPartitionEntity(OutputStream))
+	{
+		UE_LOG(LogSpatialGDKSnapshot, Error, TEXT("Error generating RoutingPartitionEntity in snapshot: %s"),
+			   UTF8_TO_TCHAR(Worker_SnapshotOutputStream_GetState(OutputStream).error_message));
```

Comment:
Would it be better to log the error message along with the code in the function where it happened? Then just log that it's gone wrong here? So that you don't need to look into other functions to see what the GetState is going to be referring to.

Reply:
Actually I guess this is following what the code around it does.

Reply:
My impression is that this is an error message that appear once you try to write anything to the snapshot stream. So the creating code does not have any error case itself.